### PR TITLE
[New Rule] Access to Browsers Credential Files

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/788

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+         "/Users/*/Library/Application Support/Firefox/Profiles/*.default/cookies.sqlite", 
+         "/Users/*/Library/Application Support/Firefox/Profiles/*.default/key*.db", 
+         "/Users/*/Library/Application Support/Firefox/Profiles/*.default/logins.json", 
+         "Login Data",
+         "Cookies.binarycookies", 
+         "key4.db", 
+         "key3.db", 
+         "logins.json", 
+         "cookies.sqlite"
+        )
```
Comment (bm11100) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:40]:
```suggestion
process where event.type in ("start", "process_started") and
  process.args :
    (
      "/Users/*/Library/Application Support/Google/Chrome/Default/Login Data", 
      "/Users/*/Library/Application Support/Google/Chrome/Default/Cookies", 
      "/Users/*/Library/Cookies*", 
      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/cookies.sqlite", 
      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/key*.db", 
      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/logins.json", 
      "Login Data",
      "Cookies.binarycookies", 
      "key4.db", 
      "key3.db", 
      "logins.json", 
      "cookies.sqlite"
    )
```

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2020/01/04"
+maturity = "production"
+updated_date = "2020/01/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a process with arguments pointing to known browser's files storing passwords and cookies.
+Adversaries may acquire credentials from web browsers by reading files specific to the target browser.
```
Comment (bm11100) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:10]:
```suggestion
Identifies the execution of a process with arguments pointing to known browser files that store passwords and cookies. Adversaries may acquire credentials from web browsers by reading files specific to the target browser.
```

Comment (bm11100) [reply rules/macos/credential_access_access_to_browser_credentials_procargs.toml:10]: may need a re-lint

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/01/04"
+maturity = "production"
+updated_date = "2020/01/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a process with arguments pointing to known browser files that store passwords and cookies. Adversaries may acquire credentials from web browsers by reading files specific to the target browser.
```
Comment (threat-punter) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:9]: This rule needs to be linted

**Review by threat-punter — APPROVED** [review]
👍

**Review by bm11100 — APPROVED** [review]

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+author = ["Elastic"]
+description = """
+Identifies the execution of a process with arguments pointing to known browser files that store passwords and cookies.
+Adversaries may acquire credentials from web browsers by reading files specific to the target browser.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Access to Browsers Credential Files"
```
Comment (bm11100) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:16]:
```suggestion
name = "Browser Credential File Access"
```

**Pull request merged**